### PR TITLE
andraszakupszki-london

### PR DESCRIPTION
**Pull request by andraszakupszki**
This pull request adds a file about London.

### CODE REVIEW

File: andraszakupszki-london.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# London, UK
+
+## Places to Eat
+- Borough Market
+- Franco Manca for :pizza:
```
Comment (andraszakupszki) [inline andraszakupszki-london.md:5]: hello

File: andraszakupszki-london.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# London, UK
+
+## Places to Eat
```
Comment (andraszakupszki) [inline andraszakupszki-london.md:3]: these are a must

File: andraszakupszki-london.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+- Borough Market
+- Franco Manca for :pizza:
+- Katzenjammers for :beers:
+
+
+## Places to Visit
```
Comment (andraszakupszki) [inline andraszakupszki-london.md]: Please finish off these

**Review by andraszakupszki — COMMENTED** [review]
Please complete these changes before merge 🚢  🚀

File: andraszakupszki-london.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# London, UK
```
Comment (CCO05948) [inline andraszakupszki-london.md:1]: Looks good to me.

File: andraszakupszki-london.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+## Places to Eat
+- Borough Market
```
Comment (CCO05948) [inline andraszakupszki-london.md:4]: What kind of food does Borough Market have?

**Review by CCO05948 — DISMISSED** [review]
@andraszakupszki please review requested changes and make any edits you feel are necessary.

File: andraszakupszki-london.md (hunk 5 above)
Comment (andraszakupszki) [reply andraszakupszki-london.md:4]: Mainly healthy/organic street food. If you're in London I strongly recommend it.

**Comment by andraszakupszki** [comment]
@CCO05948 Could you please approve?

**Pull request merged**